axum-extra/src/response/wasm.rs

```diff
@@ -0,0 +1,26 @@
+use axum::response::{IntoResponse, Response};
+use bytes::Bytes;
+use http::header;
+use http_body::Full;
+
+/// A WASM response.
+///
+/// Will automatically get `Content-Type: application/wasm`.
+#[derive(Clone, Copy, Debug)]
+#[must_use]
+pub struct Wasm<T>(pub T);
+
+impl<T> IntoResponse for Wasm<T>
+where
+    T: Into<Full<Bytes>>,
```

Comment:
Do we have the same bound in another place already? Just wondering where it comes from.

Reply:
Copied from the Html IntoResponse implementation
https://github.com/tokio-rs/axum/blob/946d8c32530ae301bf26928ab22090b33366a748/axum/src/response/mod.rs#L45-L48

Reply:
I've actually been thinking that we could make these types work for any `T: IntoResponse`. Then it wouldn't be required to have the contents in memory.

I can that in a separate PR.